lib/rubocop/cop/rake/duplicate_namespace.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rake
+      # If namespaces are defined with the same name, Rake executes the both namespaces
+      # in definition order.
+      # It is misleading sometimes. You should squash them into one definition
```

Comment:
I feel this description is a bit strange.
The description is actually true, Rake executes both namespaces in definition order.
But personally I think it is not "misleading". It is just the expected behavior, but redundant.


Actually I wrote "misleading" for duplicate task definitions because we can guess it as multiple meanings.


```ruby
task :foo do
  p 1
end

task :foo do
  p 2
end

# What does `rake foo` display?
# 1 and 2 (actual behavior), only 2 (same as def), or only 1? It is not obvious.
```

But for namespace definitions, I think we can guess all definitions are evaluated, like class and module.

So I think the description should only mention  "redundant", not "misleading". What do you think?

Reply:
I understood your explanation and fixed.
Thank you!